### ISSUE TITLE
Use mypy to check arrow.get().today() method prompt int not callable

### ISSUE THREAD

**Issue by cxyfreedom** [python/mypy#14427]
**Bug Report**

Use mypy to check `arrow.get().today()` method prompt `error: "int" not callable`

**To Reproduce**

```python
# -*- coding: utf-8 -*-
import arrow

today = arrow.get().today()
```

**Expected Behavior**

Return `Success: no issues found in 1 source file`

**Actual Behavior**

```
mypy_test.py:4: error: "int" not callable  [operator]
Found 1 error in 1 file (checked 1 source file)
```

**Your Environment**

- Mypy version used: `0.790` or `0.991`
- Mypy command-line flags: `mypy --ignore-missing-imports --check-untyped-defs mypy_test.py`
- Mypy configuration options from `mypy.ini` (and other config files):
- Python version used: `3.7.13`

**Comment by hauntsaninja** [comment]
This is because of:
https://github.com/arrow-py/arrow/blob/8842f8c3263d1f1219c189a0500aa67abdd0a214/arrow/arrow.py#L806

I'd mention it to the arrow library
